diff --git a/contrib/lease-tools/Makefile b/contrib/lease-tools/Makefile
index f38f2ed..fd62a94 100644
--- a/contrib/lease-tools/Makefile
+++ b/contrib/lease-tools/Makefile
@@ -4,3 +4,8 @@ all: dhcp_release dhcp_release6 dhcp_lease_time
 
 clean:
 	rm -f *~ *.o core dhcp_release dhcp_release6 dhcp_lease_time
+
+
+install: dhcp_release dhcp_release6 dhcp_lease_time
+	mkdir -p $(DESTDIR)/usr/bin
+	cp -a dhcp_release dhcp_release6 dhcp_lease_time $(DESTDIR)/usr/bin
