--- dnsmasq-2.80/contrib/lease-tools/Makefile~	2018-10-18 18:21:55.000000000 +0000
+++ dnsmasq-2.80/contrib/lease-tools/Makefile	2018-12-05 09:40:00.656814452 +0000
@@ -4,3 +4,8 @@
 
 clean:
 	rm -f *~ *.o core dhcp_release dhcp_release6 dhcp_lease_time
+
+
+install: dhcp_release dhcp_release6 dhcp_lease_time
+	mkdir -p $(DESTDIR)/usr/bin
+	cp -a dhcp_release dhcp_release6 dhcp_lease_time $(DESTDIR)/usr/bin
\ No newline at end of file
