From a5860f8571f4f026badda2c1befdb24040b86fc0 Mon Sep 17 00:00:00 2001
From: "Munoz, Obed N" <obed.n.munoz@intel.com>
Date: Fri, 7 Aug 2015 12:01:21 -0500
Subject: [PATCH] nov6

Signed-off-by: Munoz, Obed N <obed.n.munoz@intel.com>
---
 src/config.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/config.h b/src/config.h
index 7e4619d..0f3b088 100644
--- a/src/config.h
+++ b/src/config.h
@@ -155,7 +155,7 @@ RESOLVFILE
    has no library dependencies other than libc */
 
 #define HAVE_DHCP
-#define HAVE_DHCP6 
+#undef  HAVE_DHCP6
 #define HAVE_TFTP
 #define HAVE_SCRIPT
 #define HAVE_AUTH
-- 
2.5.0

