--- dnsmasq-2.82/src/config.h~	2020-07-19 20:54:44.000000000 +0000
+++ dnsmasq-2.82/src/config.h	2020-07-20 15:50:08.320414896 +0000
@@ -183,11 +183,11 @@
 */
 
 /* #define HAVE_LUASCRIPT */
-/* #define HAVE_DBUS */
+#define HAVE_DBUS
 /* #define HAVE_IDN */
-/* #define HAVE_LIBIDN2 */
-/* #define HAVE_CONNTRACK */
-/* #define HAVE_DNSSEC */
+#define HAVE_LIBIDN2
+#define HAVE_CONNTRACK
+#define HAVE_DNSSEC
 
 
 /* Default locations for important system files. */
