diff --git a/src/config.h b/src/config.h
index 12c2815..fab36db 100644
--- a/src/config.h
+++ b/src/config.h
@@ -186,12 +186,12 @@ RESOLVFILE
 */
 
 /* #define HAVE_LUASCRIPT */
-/* #define HAVE_DBUS */
+#define HAVE_DBUS
 /* #define HAVE_IDN */
-/* #define HAVE_LIBIDN2 */
-/* #define HAVE_CONNTRACK */
+#define HAVE_LIBIDN2
+#define HAVE_CONNTRACK
 /* #define HAVE_CRYPTOHASH */
-/* #define HAVE_DNSSEC */
+#define HAVE_DNSSEC
 
 
 /* Default locations for important system files. */
