From 5f7b39f991779280f7b898d42bef1fb714a0219f Mon Sep 17 00:00:00 2001
From: Ikey Doherty <michael.i.doherty@intel.com>
Date: Wed, 17 Jun 2015 13:48:16 +0100
Subject: [PATCH] Convert to stateless configuration

Signed-off-by: Ikey Doherty <michael.i.doherty@intel.com>
---
 src/config.h |  8 ++++++++
 src/option.c | 14 +++++++++++++-
 2 files changed, 21 insertions(+), 1 deletion(-)

diff --git a/src/config.h b/src/config.h
index f75fe9d..7e4619d 100644
--- a/src/config.h
+++ b/src/config.h
@@ -206,6 +206,14 @@ RESOLVFILE
 #   endif
 #endif
 
+#ifndef DEFAULT_CONFFILE
+#   if defined(__FreeBSD__)
+#      define DEFAULT_CONFFILE "/usr/local/share/defaults/dnsmasq/dnsmasq.conf"
+#   else
+#      define DEFAULT_CONFFILE "/usr/share/defaults/dnsmasq/dnsmasq.conf"
+#   endif
+#endif
+
 #ifndef RUNFILE
 #   if defined(__ANDROID__)
 #      define RUNFILE "/data/dnsmasq.pid"
diff --git a/src/option.c b/src/option.c
index f99c3f5..e9ced79 100644
--- a/src/option.c
+++ b/src/option.c
@@ -4492,7 +4492,19 @@ void read_opts(int argc, char **argv, char *compile_opts)
     }
   else
     {
-      one_file(CONFFILE, conffile_opt);
+#ifdef DEFAULT_CONFFILE
+      struct stat st = {0};
+      if (stat(CONFFILE, &st) != 0)
+        {
+          one_file(DEFAULT_CONFFILE, conffile_opt);
+        }
+      else
+       {
+          one_file(CONFFILE, conffile_opt);
+       }
+#else
+          one_file(CONFFILE, conffile_opt);
+#endif
     }
 
   /* port might not be known when the address is parsed - fill in here */
-- 
1.9.1

